<!--
    Date of creation: 5/II/2018
    Application created by: Ricardo Tellez <rtellez@theconstructsim.com>
    The Construct http://www.theconstructsim.com
    License LGPLV3 << Basically means you can do whatever you want with this!
-->

<launch>

    <!-- Load the parameters for the algorithm -->
    <rosparam command="load" file="$(find gripper_training)/config/qlearn_params.yaml" />

    <!-- Launch the training system -->
    <node pkg="gripper_training" name="gym_train" type="start_training.py" output="screen" respawn = "true"/>

    <!--node pkg="rqt_multiplot" name="rqt_multiplot" type="rqt_multiplot" output="screen"/-->
    <param name ="/use_sim_time" value="true"/>
</launch>
